feature/qos/otg_tests/ingress_police_nhg/README.md

```diff
@@ -0,0 +1,216 @@
+            config:
+              cir: 2000000000           # 2Gbit/sec
+              bc: 100000                 # 100 kilobytes
+              queuing-behavior: POLICE
+            exceed-action:
+              config:
+                drop: TRUE
+
+    - scheduler-policy: "limit_1Gb"
+      config:
+        name: "limit_1Gb"
+      schedulers:
+        - scheduler: 0
+          config:
+              type: ONE_RATE_TWO_COLOR
+              sequence: 0
+          one-rate-two-color:
+            config:
+              cir: 1000000000           # 1Gbit/sec
+              bc: 100000                # 100 kilobytes
+              queuing-behavior: POLICE
+            exceed-action:
+              config:
+                drop: TRUE
+  classifers:
+    - classifer: “dest_A”
+      config:
+        name: “dest_A”
+      terms:
+        - term:   # repeated for address in destination A
+          config:
+            id: "match_1_dest_A"
+          conditions:
+            next-hop-group:
+                config:
+                    name: "nhg_A"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+    - classifer: “dest_B”
+      config:
+        name: “dest_B”
+      terms:
+        - term:
+          config:
+            id: "match_1_dest_B"
+          conditions:
+            next-hop-group:
+                config:
+                    name: "nhg_B"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+
+  input-policies:       # new OC subtree input-policies (/qos/input-policies)
+    - input-policy: "limit_group_A_2Gb"
+      config:
+        name: "limit_group_A_2Gb"
+        classifer: "dest_A"
+        scheduler-policy: "limit_2Gb"
+    - input-policy: "limit_dest_group_B_1Gb"
+      config:
+        name: "limit_dest_group_B_1Gb"
+        classifer: "dest_B"
+        scheduler-policy: "limit_1Gb"
+
+  interfaces:                  # this is repeated per subinterface (vlan)
+    - interface: "PortChannel1"
+      interface-ref:
+        config:
+          subinterface: 100
+    input:
+      config:
+        policies:  [            # new OC leaf-list (/qos/interfaces/interface/input/config/policies)
+          limit_dest_group_A_2Gb
+        ]
+  interfaces:                  # this is repeated per subinterface (vlan)
+    - interface: "PortChannel1"
+      interface-ref:
+        config:
+          subinterface: 200
+    input:
+      config:
+        policies:  [            # new OC leaf-list (/qos/interfaces/interface/input/config/policies)
+          limit_dest_group_B_1Gb
+        ]
+
+```
+
+### TE-18.3.1 Test traffic
+
+* Send traffic
+  * Send traffic from ATE port 1 to DUT for dest_A and is conforming to cir.
+  * Send traffic from ATE port 1 to DUT for to dest_B and is conforming to
+    cir.
+  * Validate packets are received by ATE port 2.
+    * Validate qos interface scheduler counters
+    * Validate afts next hop counters
```

Comment:
Since we are not matching on next-hop groups, we can remove the requirement to check next-hop-group counters.